ubg/src/mafia/MafiaDay.js

```diff
@@ -1,12 +1,180 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+import Player from './Player';
+import PersonalInfo from './PersonalInfo';
+import PropTypes from 'prop-types';
+import * as firebase from 'firebase/app';
+
+const useStyles = makeStyles((theme) => ({
+  main: {
+    height: '100vh',
+  },
+  video: {
+    background: theme.palette.primary.main,
+    textAlign: '-webkit-center',
+  },
+  gameContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    position: 'relative',
+  },
+  card: {
+    width: '100%',
+  },
+  voteBtn: {
+    width: '100%',
+    alignItems: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+  },
+}));
 
 /**
  * @return {ReactElement} Mafia day element
  */
-export default function MafiaDay() {
+export default function MafiaDay({mafiaKill, doctorSave,
+  usersData, usersCollection, user, room, roomData}) {
+  const classes = useStyles();
+  const [players, setPlayers] = React.useState([]);
+  const [userInfo, setUserInfo] = React.useState('');
+  const [deathText, setDeathText] = useState('');
+  const [choice, setChoice] = useState('');
+  const [voted, setVoted] = useState(false);
+
+  /**
+   * Sets up daytime data and logistics
+   * @return {undefined}
+   */
+  function startDay() {
+    if (mafiaKill && mafiaKill.uid !== doctorSave.uid) {
+      usersCollection.doc(mafiaKill).update({alive: false});
+      setDeathText(mafiaKill.displayName + ' was killed by the mafia');
+    } else if (mafiaKill && mafiaKill.uid === doctorSave.uid) {
+      setDeathText(mafiaKill.displayName + ' was saved by the doctor');
+    }
```

Comment:
When I played, we only show "### is killed tonight" or "no one is killed tonight". I felt like if we provide explicit information would give too much information to the good side. But I'm not sure.

Reply:
Okay I'll make that change!